src/resources/cardBodyLabel.js

```diff
@@ -0,0 +1,51 @@
+const CARD_BODY_LABEL = {
+  CarePlan: 'Care Plan',
+  Condition: 'Condition',
+  Encounter: 'Encounter',
+  Immunization: 'Immunization',
+  MedicationRequest: 'Medication',
+  Procedure: 'Procedure',
+  abatement: 'Abatement',
+  age: 'Age',
+  asserted: 'Asserted',
+  claimType: 'Claim Type',
+  class: 'Class',
+  clinicalStatus: 'Clinical Status',
+  concept: 'Concept',
+  coverage: 'Coverage',
+  diagnosis: 'Diagnosis',
+  diastolic: 'Diastolic',
+  dosage: 'Dosage',
+  ending: 'Ending',
+  given: 'Given',
+  measure: 'Measure',
+  onset: 'Onset',
+  orderedBy: 'Ordered By',
+  period: 'Period',
+  primarySource: 'Primary Source',
+  provider: 'Provider',
+  quantity: 'Quantity',
+  reason: 'Reason',
+  referral: 'Referral',
+  refills: 'Refills',
+  reported: 'Reported',
+  result: 'Result',
+  role: 'Role',
+  startingOn: 'Starting On',
+  status: 'Status',
+  supply: 'Supply',
+  systolic: 'Systolic',
+  taken: 'Taken',
+  total: 'Total',
+  totalBenefit: 'Total Benefit',
+  totalCost: 'Total Cost',
+  type: 'Type',
+  use: 'Use',
+  vaccine: 'Vaccine',
+  value: 'Value',
+  valueQuantity: 'Value',
+  valueRatio: 'Value Ratio',
+  verificationStatus: 'Verification',
+};
+
```

Comment:
I wonder if we should get all these mappings of keys <==> labels in one place, a la i18n.  ...and make it easier to adapt to i18n, some day.

Reply:
Yeah, that would be ideal.